pkg/pdapi/pdapi.go

```diff
@@ -621,6 +673,24 @@ func (pc *pdClient) TransferPDLeader(memberName string) error {
 	return fmt.Errorf("failed %v to transfer pd leader to %s,error: %v", res.StatusCode, memberName, err2)
 }
 
+func (pc *pdClient) GetAutoscalingPlans(strategy Strategy) ([]Plan, error) {
```

Comment:
The construction of `strategy` will be done in another PR?

Reply:
yes